src/export_figure.py

```diff
@@ -20,7 +20,8 @@ class ExportFigureWindow(Adw.Window):
     def __init__(self, application):
         super().__init__(
             application=application, transient_for=application.main_window,
-            file_formats=application.canvas.get_supported_filetypes_grouped(),
+            file_formats=application.main_window.toast_overlay.get_child(
+            ).get_supported_filetypes_grouped(),
```

Comment:
Why this change? `get_supported_filetypes_grouped()` is a method from Matplotlibs canvas. It doesn't seem obvious why one would call it from the child of the toast overlay in the main window. (Which happens to be the same canvas)

Reply:
We no longer store a reference to the actual canvas, as this is probably (one of) the only cases, where we would need to access it.
This is why it broke before (as I didn't catch it during search).

Reply:
Ah gotcha, I didn't catch it either as I didn't think about the actual reference no longer being stored. That makes sense then.